simulator/mips/mips_instr.h

```diff
@@ -353,7 +353,7 @@ class MIPSInstr
             uint32 mask = 0x0;
             uint32 i = 0;
             for (i = 0; i < 4; ++i)                   // in this cycle we found byte from which we should start to do lwr
-                if ( (mem_addr % 4) == (v_dst >> i*8 & (mem_addr % 4))) // and fill all previous bytes with zeros
+                if ( (v_imm % 4) == (mem_addr % 4 + i) ) // and fill all previous bytes with zeros
```

Comment:
OK, and what does "v_imm" does then?
How do you test your code?

Reply:
All tests were successful

Reply:
I built tt.core.s trace locally, run it locally and all tests were successful.

Reply:
And before that I uncommented lwr instruction

Reply:
Thank you.

Please update traces submodule so that result can be reproduced on Travis/AppVeyor as well.

Reply:
```c++
uint32 i;
for (i = 0; i < 4; ++i) 
    if ( (v_imm % 4) == (mem_addr % 4 + i) )
       break;
```

I was intrigued as that code looks very strange, since it just assigns `i` to `(mem_addr - v_imm) % 4`.
However, our tests it is always zero as we do not cover cases with `v_src1 % 4 != 0`.